README.md

```diff
@@ -0,0 +1,29 @@
+# Serverless app samples for Freshworks app developers
+
+This repository contains sample apps demonstrating the usage of serverless features of the Freshworks Developer Platform.
+
+| **Product**        | | | | |
+|--------------------|-|-|-|-|
+| [Freshdesk][fd]    | [App Setup and external events][fd_app] | [Product events][fd_product] | [Scheduled events][fd_scheduled] | [Server Method Invocation][fd_smi] |
+| [Freshsales][fcrm] | - | [Product events][fcrm_product] | [Scheduled events][fcrm_scheduled] | [Server Method Invocation][fcrm_smi] |
+| [Freshservice][fs] | [App Setup and external events][fs_app] | [Product events][fs_product] | [Scheduled events][fs_scheduled] | [Server Method Invocation][fs_smi] |
+| [Freshteam][ft]    | - | [Product events][ft_product] | [Scheduled events][ft_scheduled] | [Server Method Invocation][ft_smi] |
+
+[fd]: freshdesk/
+[fd_app]: freshdesk/app-setup-and-external-events/
+[fd_product]: freshdesk/product-events/
+[fd_scheduled]: freshdesk/scheduled-events/
+[fd_smi]: freshdesk/server-method-invocation/
+[fcrm]: freshsales/
+[fcrm_product]: freshsales/product-events/
+[fcrm_scheduled]: freshsales/scheduled-events/
+[fcrm_smi]: freshsales/server-method-invocation/
+[fs]: freshservice/
+[fs_app]: freshservice/app-setup-and-external-events/
+[fs_product]: freshservice/product-events/
+[fs_scheduled]: freshservice/scheduled-events/
+[fs_smi]: freshservice/server-method-invocation/
+[ft]: freshteam/
+[ft_product]: freshteam/product-events/
+[ft_scheduled]: freshteam/scheduled-events/
+[ft_smi]: freshteam/server-method-invocation/
```

Comment:
There are some apps that are yet to be added with source code. Just coming soon readme text is there in the repo. Can we remove those apps from this list? Maybe a "Coming soon" text?